### PR TITLE
Some more Pathlib additions

### PR DESCRIPTION
**Pull request by kolibril13**
Translated some os lines into the better looking pathlib notation:
https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

### CODE REVIEW

File: tests/helpers/graphical_units.py [hunk 1]
```diff
@@ -35,9 +36,9 @@ def set_test_scene(scene_object, module_name):
     config["frame_rate"] = 15
 
     with tempfile.TemporaryDirectory() as tmpdir:
-        os.makedirs(os.path.join(tmpdir, "tex"))
-        file_writer_config["text_dir"] = os.path.join(tmpdir, "text")
-        file_writer_config["tex_dir"] = os.path.join(tmpdir, "tex")
+        os.makedirs(Path(tmpdir) / "tex")
+        file_writer_config["text_dir"] = Path(tmpdir) / "text"
+        file_writer_config["tex_dir"] = Path(tmpdir) / "tex"
```
Comment (leotrs) [inline tests/helpers/graphical_units.py:41]: In the past we have had issues with storing Path objects in file_writer_config. For now, I believe it's safer to use Path objects whenever possible, but store them as strings in the config dicts, until the config system is more stable.

**Review by leotrs — APPROVED** [review]

File: tests/helpers/graphical_units.py (hunk 1 above)
Comment (kolibril13) [reply tests/helpers/graphical_units.py]: ok, done.

**Review by leotrs — APPROVED** [review]

**Comment by huguesdevimeux** [comment]
Does this fix/is related to #485 ?

**Review by huguesdevimeux — APPROVED** [review]
SGTM

**Review by behackl — APPROVED** [review]

**Comment by leotrs** [comment]
@kolibril13 this is ready for merge, but unfortunately there are now conflicts. I can't solve the conflicts for you as they come from your fork. If you have any doubts, please let me know (as this has to do with the config system)

**Comment by kolibril13** [comment]
@leotrs : Ok, thank you!
Tomorrow, I will test if the script still does what it should do with the new config system, and if it is so, then I will merge it.

**Comment by leotrs** [comment]
What happened here?

**Comment by kolibril13** [comment]
This file `graphical_units.py` so there was a merge conflict.
Further, I guess that https://github.com/ManimCommunity/manim/pull/640 will also probably change the behaviour of graphical_units.py, so I thought it might be better to wait with this pr.

**Pull request closed without merging**